src/nl/surf/eduhub_rio_mapper/worker.clj

```diff
@@ -236,6 +238,7 @@
                   (loop []
                     (let [result (async/alt!! c                          ([v] v)
                                               (async/timeout timeout-ms) ::ping)]
+                      ;; this throws an exception if the lock has expired
```

Comment:
This should probably not raise an exception but do "best effort" and continue if extending fails